### PR TITLE
fix: remove string representations from models that do not require it

### PR DESCRIPTION
**Pull request by rtirserio**
Removes models from the update_string_representations that do not require it. This is ~going into staging~ to support the production deployment of 1.3

### CODE REVIEW

**Comment by mjoyce91** [comment]
@rtirserio could this be done to dev instead? Getting migration errors from removing old historical tables, and probably better to avoid any merge conflicts.

**Comment by burgwyn** [comment]
I agree with Mike.  Could we pair this with the feature to remove historical models?

**Comment by rtirserio** [comment]
@mjoyce91 @burgwyn - updated the target brach to dev and resolved conflicts. adding the DO NOT MERGE tag due to a demo

**Comment by rtirserio** [comment]
@mjoyce91 this is ready now that the demo has been completed

File: talentmap_api/messaging/migrations/0005_auto_20190916_1812.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('messaging', '0004_auto_20180409_1505'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='notification',
+            name='_string_representation',
```
Comment (mjoyce91) [inline talentmap_api/messaging/migrations/0005_auto_20190916_1812.py]: We don't use string representations for notifications?

Comment (rtirserio) [reply talentmap_api/messaging/migrations/0005_auto_20190916_1812.py]: the Notifications model doesnt have a `__str__` function so it uses the default which us something like `{modelName}::{instance.id}`

**Pull request merged**